docs/source/conf.py

```diff
@@ -23,10 +23,19 @@
 # -- Project information -----------------------------------------------------
 import os.path
 import sys
+from pathlib import Path
+import warnings
 
 import pytorch_sphinx_theme
 import torchrl
 
+# find project
+sys.path.insert(0, str(Path(__file__).parents[2]))
```

Comment:
instead of this, we have written a script that copies the files locally
Have a look at https://github.com/pytorch/rl/blob/49af5da9e0d21f3d3564f30c94a72d337e8de2de/docs/source/knowledge_base.py#L31-L55

If that suits your purpose, feel free to rename the method and file into something more generic (since it won't be about knowledge base anymore)